app/controllers/provider_interface/provider_users_controller.rb

```diff
@@ -33,15 +39,42 @@ def create
       redirect_to provider_interface_provider_users_path
     end
 
+    def edit_providers
+      provider_user = ProviderUser.find(params[:provider_user_id])
+      @form = ProviderUserForm.from_provider_user(provider_user)
+      @form.current_provider_user = current_provider_user
+    end
+
+    def update_providers
+      provider_user = ProviderUser.find(params[:provider_user_id])
+      provider_user.assign_attributes(provider_user_params.except(:permissions))
```

Comment:
This allows users to fiddle with the HTML and edit the user's name and email address.